app/signin/_components/SigninButton.tsx

```diff
@@ -1,12 +1,12 @@
 import { MouseEventHandler, ReactNode } from 'react';
 
-import Image from 'next/image';
+import Image, { StaticImageData } from 'next/image';
 
 import styles from './SigninButton.module.scss';
 
 interface Props {
```

Comment:
```suggestion
interface SigninButtonProps {
```
컴포넌트의 프롭스타입은 컴포넌트 이름을 붙이기로 하지 않았었나요?!(질문)

Reply:
앗~ 깜빡했습니다😂